tools/scripts-config-eslint/.eslintrc.json

```diff
@@ -88,7 +88,9 @@
     "testing-library/custom-renders": "off",
     "testing-library/custom-queries": "off",
     "testing-library/no-node-access": "off",
-    "testing-library/render-result-naming-convention": "off"
+    "testing-library/render-result-naming-convention": "off",
+
```

Comment:
why this empty line ?

Reply:
To be homogeneous with existing format for rules : there is a paragraph for each "rule provider" (see above)